types/lib.d.ts

```diff
@@ -153,6 +153,13 @@ declare module 'stripe' {
       host?: string;
     }
 
+    export type RawRequestOptions = RequestOptions & {
```

Comment:
Doesnt `RequestOptions` already have headers?

Reply:
Doesn't look like it: https://github.com/stripe/stripe-node/blob/2a2fee03d9c43f671445f1a27006b34c53467dbb/types/lib.d.ts#L117-L154

Reply:
Ah my bad! I was looking at the type definition for a similarly named RequestOptions interface which happens to be internal only